__tests__/app.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require("supertest")
+const db = require("../db/connection")
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data")
+
+beforeEach(() => seed(data))
+afterAll(() => db.end())
+
+describe("/api/topics", () => {
+    test(":) GET 200 - returns an array of topics with properties slug and describtion", () => {
+        return request(app).get("/api/topics").expect(200).then(({body}) => {
+            expect(body.length).not.toBe(0)
+            body.forEach((topic) => {
+                expect(topic).toEqual({
+                    "slug": expect.any(String),
+                    "description": expect.any(String)
+                })
+            })
+        })
+    })
+    test(":( GET 404 - returns route not found for invalid end-points", () => {
```

Comment:
As this 404 error applies to all endpoints, it would be better to separate this out into its own describe block.